sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -65,19 +65,19 @@ object ScalaJSPluginInternal {
       "Scala.js internal: Whether a project really wants the DOM. " +
       "Calculated using requiresDOM and jsDependencies", KeyRanks.Invisible)
 
-  /** Default post link environment */
-  val scalaJSDefaultPostLinkJSEnv = TaskKey[JSEnv]("scalaJSDefaultPostLinkJSEnv",
-      "Scala.js internal: Default for postLinkJSEnv", KeyRanks.Invisible)
-
-  /** Lookup key for CompleteClasspath in attribute maps */
-  val scalaJSCompleteClasspath =
-      AttributeKey[CompleteClasspath]("scalaJSCompleteClasspath")
-
   /** All .sjsir files on the fullClasspath, used by scalajsp. */
-  val sjsirFilesOnClasspath = TaskKey[List[String]]("sjsirFilesOnClasspath",
+  val sjsirFilesOnClasspath = TaskKey[Seq[String]]("sjsirFilesOnClasspath",
       "All .sjsir files on the fullClasspath, used by scalajsp",
       KeyRanks.Invisible)
 
+  val sourceFiles = AttributeKey[Seq[File]]("sourceFiles",
+      "Files used to this value (can be used in FileFunctions later).", KeyRanks.Invisible)
```

Comment:
The name of this attribute key is very likely to clash with other sbt plugins. Consider renaming it to `scalaJSSourceFiles`.